### PR TITLE
Add wrap files for libsrtp2-2.2.0

### PR DESCRIPTION
**Pull request by xhaakon**

### CODE REVIEW

File: crypto/test/meson.build [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+includedirs = include_directories(
+  '../..',
+  '../../include',
+  '../../test',
+  '../../crypto/include',
+)
+
+foreach t : tests
+test(t[0],
```
Comment (sarum9in) [inline crypto/test/meson.build]: Looks like indentation is broken here.

File: upstream.wrap [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = libsrtp2-2.2.0
```
Comment (sarum9in) [inline upstream.wrap]:
```
$ tar tf v2.2.0.tar.gz 
libsrtp-2.2.0/
```

The directory is called `libsrtp-2.2.0`.

File: crypto/test/meson.build (hunk 1 above)
Comment (xhaakon) [reply crypto/test/meson.build]: Fixed.

File: upstream.wrap (hunk 2 above)
Comment (xhaakon) [reply upstream.wrap]: ...and fixed.

**Pull request merged**